README.md

```diff
@@ -26,7 +26,7 @@ If you need support, start with the [user documentation](https://cloudinit.readt
 
 If you need additional help consider reaching out with one of the following options:
 
-- Ask a question in the [``#cloud-init`` IRC channel on Freenode](https://webchat.freenode.net/?channel=#cloud-init)
+- Ask a question in the [``#cloud-init`` IRC channel on Libera](https://libera.chat/)
```

Comment:
is there an equiv webui direct link? Or direct link that might trigger an xdg-open of the irc client for a user? If not all good, but if so seems like it might be a more equiv. update.

Reply:
Checked last night and couldn't find anything, but just checked again and updated it.

Reply:
:+1: